packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -168,34 +218,22 @@ export class EthereumAnchorService implements AnchorService {
       sendRequest$ = sendRequest$.pipe(
         retry({
           delay: (error) => {
-            this._logger.err(
-              new Error(
-                `Error connecting to CAS while attempting to anchor ${carFileReader.streamId} at commit ${carFileReader.tip}: ${error.message}`
-              )
+            this.#logger.err(
```

Comment:
Since we're retrying, this should probably be a warning saying it failed and we are retrying. 

Only log an error if we give up on trying to send it. 

We also might want to log and error when we shutdown for these pending requests.